doc/source/whatsnew/v0.21.0.txt

```diff
@@ -583,6 +583,12 @@ PyPy
 - Fix :func:`DataFrame.memory_usage` to support PyPy. Objects on PyPy do not have a fixed size,
   so an approximation is used instead (:issue:`17228`)
 
+Documentation
+^^^^^^^^^^^^^
+
+- Improved docs for documentation contributors (:issue:`17579`)
+
+
```

Comment:
Not necessary in this case.  Generally documentation changes don't get a `whatsnew`.

Reply:
Why not? They're a contribution, and it demonstrates the project values a documentation contributor. :smile:

Reply:
I think our documentation speaks for itself when it comes to how much we value it. :smile:

That being said, our change-logs are generally for functional changes / bug fixes.

Reply:
@gfyoung I believe that you value it :smile:

My point is that the cost of putting documentation changes at the end of the what's new are outweighed by the benefits of encouraging contributors to the documentation.

Reply:
@willingc : Still would like to be removed.  As I said, we generally don't log documentation changes.

Reply:
In the past that’s what we’ve done, but I’m open to logging doc changes here. I could imagine someone reading the change log and seeing that a section has been rewritten and clicking through to see what’s changed.